samples/matter/smoke_co_alarm/sample.yaml

```diff
@@ -57,17 +57,20 @@ tests:
     tags:
       - sysbuild
       - ci_samples_matter
-  sample.matter.smoke_co_alarm.release.nrf54l15.power_consumption:
+  sample.matter.smoke_co_alarm.release.power_consumption:
     sysbuild: true
     build_only: true
     extra_args:
       - FILE_SUFFIX=release
-      - CONFIG_NCS_SAMPLE_MATTER_LEDS=n
-      - CONFIG_NCS_SAMPLE_MATTER_WATCHDOG=n
+      - smoke_co_alarm_SNIPPET=power-consumption-tests
     integration_platforms:
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpuapp
       - nrf54l15dk/nrf54l15/cpuapp
       - nrf54l15dk/nrf54l10/cpuapp
```

Comment:
do we want to remove nrf54l10 platform?

Reply:
I can do that, if you want. I've just extended the existing configuration that included L10 as well.

Reply:
lets remove nrf54l10 from that configuration